src/components/ExploreGallery/styled.js

```diff
@@ -0,0 +1,7 @@
+import styled from "styled-components";
+import Cursor from "../../images/cursor-pointer.svg";
+
+export const ViewContainer = styled.div`
+  margin-bottom: 6rem;
```

Comment:
i'm not sure how to help with this but the gallery seems to overflow out of the view container so the margin at the bottom is still really tight

Reply:
I set a height and set overflow hidden on the container and it fixed the bottom margin :)